docs/api/classifier/prcurve.rst

```diff
@@ -64,24 +65,36 @@ To support multi-label classification, the estimator is wrapped in a `OneVsRestC
     viz.score(X_test, y_test)
     viz.poof()
 
-.. image:: images/multiclass_precision_recall.png
 
 A more complex Precision-Recall curve can be computed, however, displaying the each curve individually, along with F1-score ISO curves (e.g. that show the relationship between precision and recall for various F1 scores).
 
-.. code:: python
+.. plot::
+    :context: close-figs
+    :alt: PrecisionRecallCurves displaying each curve individually
 
     from sklearn.naive_bayes import MultinomialNB
+    from sklearn.preprocessing import LabelEncoder, OrdinalEncoder
+    from sklearn.model_selection import train_test_split as tts
+    from yellowbrick.classifier import PrecisionRecallCurve
+    from yellowbrick.datasets import load_game
 
-    oz = PrecisionRecallCurve(
+    # Load dataset and encode categorical variables
+    X, y = load_game()
+    X = OrdinalEncoder().fit_transform(X)
+    y = LabelEncoder().fit_transform(y)
+
+    X_train, X_test, y_train, y_test = tts(X, y, test_size=0.2, shuffle=True)
+
+    # Create the visualizer, fit, score, and poof it
+
+    viz = PrecisionRecallCurve(
         MultinomialNB(), per_class=True, iso_f1_curves=True,
         fill_area=False, micro=False
```

Comment:
@lwgray now that #688 has been merged in, would you mind updating the code as follows:

```python
encoder = LabelEncoder()
y = encoder.fit_transform(y)

oz = PrecisionRecallCurve(
    MultinomialNB(), per_class=True, iso_f1_curves=True, fill_area=False, 
    micro=False, classes=encoder.classes_
)
```

This will be a good test to ensure the bug is fixed and is potentially a more realistic example! Thank you and @zjpoh for this enhancement!

Reply:
done!
![prcurve-3](https://user-images.githubusercontent.com/1000117/52830276-1eba4280-309e-11e9-887c-df488b5aca89.PNG)

Reply:
thank you!